Mac: Errors when installing pip libraries via setup

When invoking `pip install -e . `to run the `setup.py` file, we get an error:

`AssertionError: You must first install pyobjc-core and pyobjc: https://pyautogui.readthedocs.io/en/latest/install.html `

However, we have already added `pyobjc` and `pyobjc-core ` in the setup file, so it should be available.

The workaround is to install `pyobjc` and `pyobjc-core ` individually before running `pip install -e .`, but we'd like to not have to do this.

--- a/bootstrap/linux_bootstrap.sh
+++ b/bootstrap/linux_bootstrap.sh
@@ -35,11 +35,6 @@ sudo apt-get -y install \
 sudo apt-get -y install virtualenv \
 	|| sudo apt-get -y install python-virtualenv
 
-# sudo python -m pip install --upgrade --force pip
-
-# Installing xlib here to avoid conflicts with other libraries we install via setup.py
-pip install xlib
-
 # Install Leptonica
 cd ~
 wget http://www.leptonica.com/source/leptonica-1.75.3.tar.gz

--- a/bootstrap/osx_bootstrap.sh
+++ b/bootstrap/osx_bootstrap.sh
@@ -4,9 +4,3 @@
 brew install tesseract
 brew install p7zip
 brew cask install xquartz
-
-# Installing these here to avoid conflicts with other things
-# we install in setup.py
-
-pip install pyobjc
-pip install pyobjc-core

--- a/setup.py
+++ b/setup.py
@@ -8,9 +8,9 @@ PACKAGE_NAME = 'iris'
 PACKAGE_VERSION = '0.1'
 
 INSTALL_REQUIRES = [
-    # 'xlib==0.21;platform_system=="Linux"',
-    # 'pyobjc-core;platform_system=="Darwin"',
-    # 'pyobjc;platform_system=="Darwin"',
+    'xlib==0.21;platform_system=="Linux"',
+    'pyobjc-core;platform_system=="Darwin"',
+    'pyobjc;platform_system=="Darwin"',
     'pyobjc-framework-Quartz;platform_system=="Darwin"',
     'coloredlogs',
     'hashfs',
@@ -19,7 +19,7 @@ INSTALL_REQUIRES = [
     'pytesseract==0.2.0',
     'numpy==1.14.2',
     'image',
-    'PyAutoGUI==0.9.36',
+    'PyAutoGUI==0.9.38',
     'pyperclip==1.6.0',
     'packaging',
     'GitPython',

--- a/setup.py
+++ b/setup.py
@@ -9,9 +9,9 @@ PACKAGE_VERSION = '0.1'
 
 INSTALL_REQUIRES = [
     'xlib==0.21;platform_system=="Linux"',
-    'pyobjc-core;platform_system=="Darwin"',
-    'pyobjc;platform_system=="Darwin"',
-    'pyobjc-framework-Quartz;platform_system=="Darwin"',
+    'pyobjc-core==5.0;platform_system=="Darwin"',
+    'pyobjc==5.0;platform_system=="Darwin"',
+    'pyobjc-framework-Quartz==5.0;platform_system=="Darwin"',
     'coloredlogs',
     'hashfs',
     'python-dateutil',

